doc/how_to_guides/pilz_industrial_motion_planner/pilz_industrial_motion_planner.rst

```diff
@@ -397,13 +565,83 @@ constraints of an individual ``moveit_msgs::msg::MotionPlanRequest`` are
 already satisfied but the ``MoveGroupSequenceAction`` capability doesn't
 implement such a check to allow moving on a circular or comparable path.
 
-See the ``pilz_robot_programming`` package for a `ROS 1 Python script
-<https://github.com/PilzDE/pilz_industrial_motion/blob/melodic-devel/pilz_robot_programming/examples/demo_program.py>`_
-that shows how to use the capability.
+The action client needs to be initialized:
 
-Service interface
-~~~~~~~~~~~~~~~~~
+::
 
-The service ``plan_sequence_path`` allows the user to generate a joint
-trajectory for a ``moveit_msgs::msg::MotionSequenceRequest``.
-The trajectory is returned and not executed.
+    // MoveGroupSequence action client
+    using MoveGroupSequence = moveit_msgs::action::MoveGroupSequence;
+    auto action_client = rclcpp_action::create_client<MoveGroupSequence>(node, "/sequence_move_group");
+
+    // Verify that the action server is up and running
+    if (!action_client->wait_for_action_server(std::chrono::seconds(10)))
+    {
+       RCLCPP_ERROR(LOGGER, "Error waiting for action server /sequence_move_group");
+       return -1;
+    }
+
+Then, the request is created:
+
+::
+
+    // Create a MotionSequenceRequest
+    moveit_msgs::msg::MotionSequenceRequest sequence_request;
+    sequence_request.items.push_back(item1);
+    sequence_request.items.push_back(item2);
+
+Create goal and planning options. A goal response callback and result callback can be included as well.
```

Comment:
Same here: "Create goal and planning options" isn't quite a full sentence, could you reword?

Reply:
Something simple like "The goal and planning options are configured." ?